app/assets/stylesheets/application.css.scss

```
@@ -1,7 +1,8 @@
-$font-path: '/fonts';
-$image-path: '';
-
-@import 'identity-style-guide/dist/assets/scss/styles';
+@forward 'uswds' with(
+  $theme-font-path: '/fonts',
+  $theme-image-path: '',
+  );
```

idk what the typical SASS style is but for any other language I would unindent the last paren:
```suggestion
@forward 'uswds' with(
  $theme-font-path: '/fonts',
  $theme-image-path: '',
);
```

---

updated in https://github.com/18F/identity-dashboard/pull/748/commits/49d6e83970cd1f9fe1118eb7dda8770c84763753